go.mod

```diff
@@ -190,6 +190,7 @@ require (
 replace (
 	// temporary replacement for development purposes
 	github.com/containerd/containerd => github.com/containerd/containerd v1.5.0
```

Comment:
Looks like we are pinning containerd to an older version. Can we do one of the following?

- Remove the replace clause here, OR
- Update the containerd version to 1.6.1 so the require and replace versions are in sync.

:crossed_fingers: this avoids the explict replace dependency on imgcrypt.

Reply:
updated